.github/workflows/jl_test.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        jl_version: ["1.3", "1.6", "1.8"]
+        jl_version: ["1.4", "1.6", "1.8"]
```

Comment:
Testing on Julia 1.3 leads to 

![image](https://user-images.githubusercontent.com/6675409/204405343-a833ab4d-5025-4396-a1a3-14f39f75645b.png)

We should probably update

https://github.com/plotly/Dash.jl/blob/f0606e07d2479fd8b5be1da4a6a59656e24acfa3/Project.toml#L39

to match the [PlotlyBase.jl compat](https://github.com/sglyon/PlotlyBase.jl/blob/72cfaf977edaaa9f51c7264810c6e44ed4864bb2/Project.toml#L27)